UI/window-basic-filters.cpp

```diff
@@ -1169,18 +1165,16 @@ void OBSBasicFilters::FilterNameEdited(QWidget *editor, QListWidget *list)
 	editActive = false;
 }
 
-void OBSBasicFilters::AsyncFilterNameEdited(
-	QWidget *editor, QAbstractItemDelegate::EndEditHint endHint)
+void OBSBasicFilters::AsyncFilterNameEdited(QWidget *editor,
+					    QAbstractItemDelegate::EndEditHint)
 {
 	FilterNameEdited(editor, ui->asyncFilters);
-	UNUSED_PARAMETER(endHint);
 }
 
-void OBSBasicFilters::EffectFilterNameEdited(
-	QWidget *editor, QAbstractItemDelegate::EndEditHint endHint)
+void OBSBasicFilters::EffectFilterNameEdited(QWidget *editor,
+					     QAbstractItemDelegate::EndEditHint)
```

Comment:
`QAbstractItemDelegate::EndEditHint` can be removed

Reply:
Doesn't work. As removing parameters only works with auto generated slots.

Reply:
See above.

Reply:
Fixed